sql/core/src/test/scala/org/apache/spark/sql/FilterPushdownBenchmark.scala

```diff
@@ -32,14 +32,14 @@ import org.apache.spark.util.{Benchmark, Utils}
  */
 object FilterPushdownBenchmark {
   val conf = new SparkConf()
-  conf.set("orc.compression", "snappy")
-  conf.set("spark.sql.parquet.compression.codec", "snappy")
+    .setMaster("local[1]")
+    .setAppName("FilterPushdownBenchmark")
+    .set("spark.driver.memory", "3g")
```

Comment:
these and master - change to setIfMissing()? I think it's great if these can be set via config

Reply:
aha, ok. Looks good to me.
I just added this along with other benchmark code, e.g., `TPCDSQueryBenchmark`.
If no problem, I'll fix the other places in follow-up.